crates/zed/src/zed.rs

```diff
@@ -701,24 +764,6 @@ fn register_actions(
                 open_telemetry_log_file(workspace, window, cx);
             },
         )
-        .register_action(
-            move |_: &mut Workspace, _: &zed_actions::OpenKeymap, window, cx| {
```

Comment:
I do not understand how this lines up with your "we cannot remove from custom workspace actions" idea.
Also, this does not remove then newly `OpenDefaultKeymap` action added above in the latest commit.